File: Pemrograman1/Modul2/C/PRAK205-2310817310009-MuhammadFauzanAhsani.c
#include <stdio.h>
#include <math.h>

void main(){
    double a, b ;
    scanf("%lf %lf", &a, &b);
    double c = sqrt(pow(b, 2) - pow(a, 2));
    printf("Alas = %.0lf cm\n", c);
    printf("Tinggi = %.0lf cm\n", a );
    printf("Keliling = %.0lf cm\n", a + b + c );
    printf("Luas = %.0lf cm^2\n", .5 * c * a );
}

File: Pemrograman1/Modul2/Python/PRAK205-2310817310009-MuhammadFauzanAhsani.py
import math

a, b = input().split()
a, b = map(float, (a, b))
c = math.sqrt(b**2 - a**2)

print(f"Alas = {c:.0f} cm")
print(f"Tinggi = {a:.0f} cm")
print(f"Keliling = {a+b+c:.0f} cm")
print(f"Luas = {.5 * a * c:.0f} cm^2")
